connectors/cosmos/connector.go

```diff
@@ -189,6 +199,19 @@ func (cc *Connector) Setup(ctx context.Context, t iface.Transport) error {
 	}
 
 	slog.Info("Cosmos Connector has been configured with ID " + (string)(cc.id))
+	
```

Comment:
Don't leave these as comments please. Either remove them, or use slog.Debug - for simplicity, you could do `slog.Debug(fmt.Sprintf("Connector config: %+v", cc.settings))`